java/code/src/com/redhat/rhn/manager/kickstart/KickstartUrlHelper.java

```diff
@@ -350,13 +350,14 @@ public static String getCobblerProfilePath(String profileName) {
     /**
      * Get the cobbler profile url
      * @param data the kickstart data
+     * @param userFacing whether the URL is to be used by the user or just internally used.
      * @return the url
      */
-    public static String getCobblerProfileUrl(KickstartData data) {
+    public static String getCobblerProfileUrl(KickstartData data, boolean userFacing) {
```

Comment:
Just a nitpick thought, but wouldn't creating a new method be a little more readable/explaining?

Like `getProfileRenderUrl` which would use Cobbler Host and `getProfileDownloadUrl` for Java hostname?

Reply:
Addressed, with different naming